security/azure/restrict_rdp_internet/README.md

```diff
@@ -0,0 +1,36 @@
+# Azure Network Security Groups with inbound 3389 open
+
+## What it does
+
+This policy checks all azure subscriptions for Network Security Groups that have 3389 open to the internet.
```

Comment:
Azure should be capitalized.
There should be comment along the lines of "An incident is raised with the offending Network Security Groups if any are found with port 3389 open."